### ISSUE TITLE
'Symbol' is undefined in IE after using babel Core Latest Version 3 : For 7597 Need Solution

### ISSUE THREAD

**Issue by qadirfarooq** [babel/babel#13147]
from the issue # 7597 i never get what the solution is to this problem. i added latest @babel/Core my Babel version is 7.6.0. in depth list and Bebel config is given below. none of the configuration is working for IE 11. getting Symbol is undefined issue.
please help me as the project is stopped working in IE. i am new to all this so would appreciate if some one can be specific and let me know what is going wrong and where.
module.exports = {
  presets: [
     "@babel/preset-react",
      ["@babel/preset-env", {
        "targets": {
          "browsers": [
            "last 2 versions",
            "ie >= 11"
          ]
        },
        useBuiltIns: "entry", // or "entry"
   
      }]
  ],
  plugins: [
  ["@babel/plugin-transform-runtime"],
  ["@babel/plugin-transform-for-of", {
        "assumeArray": false // defaults to false
      }]
 
  ],
};
  
├── @babel/core@7.13.15
├── @babel/plugin-syntax-jsx@7.12.13
├── @babel/plugin-transform-for-of@7.13.0
├── @babel/plugin-transform-runtime@7.13.15
├── @babel/polyfill@7.12.1
├── @babel/preset-react@7.13.13
├── @babel/runtime-corejs3@7.13.10
├── @saeris/vue-spinners@1.0.8
├── @vue/cli-plugin-babel@3.12.1
├── @vue/cli-service@4.5.12
├── axios@0.19.2
├── babel-loader@8.2.2
├── babel-polyfill@6.26.0
├── bootstrap-vue@2.21.2
├── core-js@3.10.1
├── corejs@1.0.0
├── es6-promise@4.2.8
├── es6-symbol@3.1.3
├── idle-vue@2.0.5
├── jspdf-autotable@3.5.14
├── jspdf@2.3.1
├── lodash@4.17.21
├── moment@2.29.1
├── node-sass@4.14.1
├── query-string@5.1.1
├── regenerator-runtime@0.13.7
├── sass-loader@7.3.1
├── this@1.1.0
├── try@0.13.3
├── tslib@2.2.0
├── url-search-params-polyfill@8.1.1
├── v-select@0.0.1
├── vue-bootstrap-typeahead@0.2.6
├── vue-image-lightbox-carousel@1.0.7
├── vue-js-toggle-button@1.3.3
├── vue-json-csv@1.2.12
├── vue-json-excel@0.2.99
├── vue-lazyload@1.3.3
├── vue-moment@4.1.0
├── vue-pagination-2@2.0.3
├── vue-popperjs@2.3.0
├── vue-radio-toggle-buttons@0.0.5
├── vue-router@3.5.1
├── vue-select@3.11.2
├── vue-tables-2@2.2.1
├── vue-template-compiler@2.6.12
├── vue-upload-multiple-image@1.1.6
├── vue@2.6.12
├── vue2-dropzone@3.6.0
├── vuejs-datepicker@1.6.2
├── vuelidate@0.7.6
└── vuex@3.6.2

**Comment by JLHwung** [comment]
Did https://github.com/babel/babel/issues/11763#issuecomment-651792112 solve this question?

**Comment by qadirfarooq** [comment]
no. i tried this now.. still same issue. here is my config file.

module.exports = {
  presets: [
     ***@***.***/preset-react",
      ***@***.***/preset-env", {
        "targets": {
          "browsers": [
            "last 2 versions",
            "ie >= 11"
          ]
        },
        useBuiltIns: "entry", // or "entry"

      }]
  ],
  plugins: [
  ***@***.***/plugin-transform-runtime"],
  ***@***.***/plugin-transform-for-of", {
        //"loose": true, // defaults to false
        //"assumeArray": false, // defaults to false,
        "allowArrayLike": true
      }]

  ],
};

**Comment by JLHwung** [comment]
Can you provide a reproducing repo?

**Comment by qadirfarooq** [comment]
well i have to somehow take all my modules out and create one for you. will
do that today or tomorrow and let you know. thanks for all your help. will
update you soon.

**Comment by qadirfarooq** [comment]
Hi,

sorry for the delay. I was busy with some other projects. Please open the
link thats  a test web FE. please download and open in VS code and run Npm
install to get the npm module. Once you have all that done you will see the
login page.
you will see the app can be open in chrome but not in IE 11. let me know if
you have any other questions. much appreciated. as i am new so i might be
doing something wrong.

https://github.com/qadirfarooq/testApp/tree/main

Thanks,
Muhammad

**Comment by qadirfarooq** [comment]
Any updates on this?

**Comment by fengyuanchen** [comment]
In my case, I find the error comes from a third-party package: [`query-string@8.1.0`](https://github.com/sindresorhus/query-string/blob/c5c2efc5addf0d05a6269da4ef2cb365dd1b9e2c/base.js#LL10C34-L10C34).
